File: codeforces/low_difficulty/difficulty_800/59.choosing_teams_[432A]/choosing_teams.py
# Author: José Rodolfo (jric2002)
import math
n, k = list(map(int, input().split(" ")))
max_num_teams = 0
counter = 0
y = [int(number) for number in input().split(" ")]
for i in range(0, n):
  if ((5 - y[i]) >= k):
    counter += 1
max_num_teams = math.floor(counter / 3)
print(max_num_teams)

File: codeforces/low_difficulty/difficulty_800/59.choosing_teams_[432A]/choosing_teams.cpp
#include <iostream>
#include <cmath>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n, k, y, max_num_teams;
  unsigned short int counter = 0;
  cin >> n >> k;
  for (unsigned short int i = 0; i < n; i++) {
    cin >> y;
    if ((5 - y) >= k) {
      counter += 1;
    }
  }
  max_num_teams = floor(counter / 3);
  cout << max_num_teams << endl;
  return 0;
}